### DISCUSSION TITLE
aud parameter as string

### DISCUSSION THREAD

**Discussion by sidharthramesh** [ory/hydra#3002] [Q&A] [answered]
Currently, Hydra outputs an array for the "aud" parameter in the user token. Is it possible to output a string instead? There is only 1 audience, and it's the client_id. This is needed to maintain compatibility with Okta for a project I'm working on.

**Comment by vinckr** [comment, marked as answer]
Hello @sidharthramesh 
So I found this discussion: https://github.com/ory/hydra/issues/790#issuecomment-418960335
and this one https://github.com/ory/hydra/issues/883.
I am not sure but you probably need some middleware to work around it, is that possible?

**Comment by sidharthramesh** [comment]
For now, I’ve made changes to the library that does the audience validation to look for an array instead of a string. However, not all implementations work this way. Notably the okta client js library.

By middleware you mean a proxy that can relay the requests back with the intended format?

**Reply by vinckr** [reply]
>By middleware you mean a proxy that can relay the requests back with the intended format?

Yes 👍
